### PR TITLE
svtplay-dl 0.20.2015.10.08

### PR DESCRIPTION
**Pull request by spaam**
new version of svtplay-dl :tada:

### CODE REVIEW

File: Library/Formula/svtplay-dl.rb [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
+  end
+
   def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    resources.each do |r|
+      r.stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
     bin.install "svtplay-dl"
```
Comment (DomT4) [inline Library/Formula/svtplay-dl.rb]: Presumably we need to env script this so it can actually find the `requests` vendored?

Comment (spaam) [inline Library/Formula/svtplay-dl.rb]: is it better to use?
```
    ENV.prepend_create_path "PYTHONPATH", libexec+"lib/python2.7/site-packages"

    resources.each do |r|
      r.stage { system "python", *Language::Python.setup_install_args(libexec) }
    end
```

Comment (DomT4) [inline Library/Formula/svtplay-dl.rb]: You should be able to do something like:

```ruby
(libexec/"bin").install "svtplay-dl"
bin.install Dir["#{libexec}/bin/*"]
bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
```

**Comment by DomT4** [comment]
Closing as dead. Please feel free to resubmit as fresh with the comments above addressed and audit passing, if you'd like to. Thanks for the submission to Homebrew!

**Pull request closed without merging**